apps/etterlatte-behandling/src/main/kotlin/vilkaarsvurdering/klienter/GrunnlagKlientVV.kt

```diff
@@ -16,7 +16,7 @@ import no.nav.etterlatte.libs.ktor.token.BrukerTokenInfo
 import org.slf4j.LoggerFactory
 import java.util.UUID
 
-interface GrunnlagKlient {
+interface GrunnlagKlientVV {
```

Comment:
Ah, denne er berre flytta over frå vv-appen til behandling, tok meg eit par sekund å skjønne. Absolutt potensiale for å forenkle her etter kvart, ja.

Reply:
Hehe ja absolutt, men kan legge inn kommentar om det. Men det var for å spare meg for flere endringer samtidig :D